_posts/2018-04-27-pulp_ansible-beta-release.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Pulp Ansible Plugin Beta 1 Release
+author: David Davis
+tags:
+  - release
+  - 3.0
+  - pulp_ansible
```

Comment:
This tag won't work. You have to do a few small things to 'add a tag' See this dir if you want to do that:  https://github.com/pulp/pulpproject.org/tree/gh-pages/tags